src/View/View.php

```diff
@@ -796,7 +796,7 @@ public function set($name, $value = null)
     {
         if (is_array($name)) {
             if (is_array($value)) {
-                $data = array_combine($name, $value);
+                $data = array_combine($name, $value) ?: [];
```

Comment:
We probably should throw an exception here since it's a public method and user could mistakenly provide array with different number of values in each array.